debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/FindModulesCommand.java

```diff
@@ -105,29 +103,24 @@ public void run(String command, String[] args, Context context, PrintStream out)
 		
 		try {
 			int result;
-			switch (status) {
+			switch (subcommand) {
 			case MODULE:
 				result = ModularityHelper.iterateModules(out, moduleFilter, moduleOutput, filterArg);
-				out.printf("Found %d module%s\n", result, (1 == result ? "": "s"));
+				out.printf("Found %d module%s%n", result, (1 == result ? "": "s"));
 				break;
 			case PACKAGE:
 				/* Output the module(s) owning all packages matched by packageFilter */
 				ModularityHelper.PackageOutput packageOutput = ModularityHelper::printPackageJ9Module;
 				result = ModularityHelper.iteratePackages(out, packageFilter, packageOutput, filterArg);
-				out.printf("Found %d module%s\n", result, (1 == result ? "": "s"));
-				break;
-			case HELP:
```

Comment:
I think this case should stay: if the user asked for help, the response should be different than the error case.

Reply:
I will return this and add a similar case in `!dumpmodule` for consistency then.